arch/arc/core/regular_irq.S

```diff
@@ -112,11 +112,6 @@ SECTION_FUNC(TEXT, _rirq_exit)
 	mov r1, _kernel
 	ld_s r2, [r1, _kernel_offset_to_current]
 
-	/*
-	 * Lock interrupts to ensure kernel queues do not change from this
```

Comment:
Agree that this comment can be removed, not fully sure about your commit message rationale, though. It says "as here no disabling of interrupts happens any longer", though interrupts are disabled at the start of _rirq_exit().  Guess with SMP, the 'lock interrupt to ensure kernel queues do not change' part doesn't hold anymore, so simply removing all of this makes sense.

Reply:
@ruuddw I meant that in this particular location we don't disable IRQs any longer as they are now disabled much earlier. So maybe my wording was not very clean.

Reply:
Are you going to reword this, @abrodkin ?

Reply:
@ioannisg Sure may do it right away. Give me 5 mins.